src/ashley/permissions.py

```diff
@@ -55,3 +55,17 @@ def perform_permissions_check(self, user, obj, perms):
     def has_object_permission(self, request, view, obj):
         """Check user has permission `can_manage_moderator` for first forum in this LTIContext"""
         return self._can(request)
+
+
+class AuthenticatedPermission(PermissionRequiredMixin):
+    """Permission checking if user is authenticated"""
+
+    # pylint: disable=R0201,unused-argument
+    def has_permission(self, request, view):
+        """Check user is authenticated"""
+        return request.user.is_authenticated
+
+    # pylint: disable=R0201,unused-argument
+    def has_object_permission(self, request, view, obj):
+        """Check user is authenticated"""
+        return request.user.is_authenticated
```

Comment:
Django Rest Framework provides a permission class called `IsAuthenticated`.
Unless I missed something, you should use it instead of this class that does the same.